src/js/geodash/GraphWidget.js

```diff
@@ -111,7 +111,7 @@ export default class GraphWidget extends React.Component {
 
   loadTimeSeries = () => {
     const { widget, plotExtentPolygon } = this.props;
-    const path = this.widgetIsCustom() ? "timeSeriesByAsset" : "timeSeriesByIndex";
+    const path = this.widgetIsCustom() ? "timeSeriesByIndex" : "timeSeriesByIndex"; // how to remove this line and have fn work still..?
```

Comment:
I'd say just set the path as the `timeSeriesByIndex`, like you've mentioned in your comment